clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3944,10 +3944,13 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   // Check number of inputs for sanity. We need at least one input.
   assert(Inputs.size() >= 1 && "Must have at least one input.");
   // CUDA/HIP compilation may have multiple inputs (source file + results of
-  // device-side compilations). OpenMP and SYCL device jobs also take the host
-  // IR as a second input. Module precompilation accepts a list of header files
-  // to include as part of the module. All other jobs are expected to have
-  // exactly one input.
+  // device-side compilations).
+  // OpenMP device jobs take the host IR as a second input.
+  // SYCL host jobs accept the integration header from the device-side
```

Comment:
Should it be mentioned that the second input for SYCL is the integration header?

Reply:
That was my intention, at least. Is it the new wording that causes the confusion?

Reply:
Maybe something like `SYCL host jobs accept the integration header from the device-side compilation as the second input.`

Reply:
Sounds good, updated